### DISCUSSION TITLE
filter datafield that contains multiple links

### DISCUSSION THREAD

**Discussion by louwelyn-jeremy** [marcusolsson/obsidian-projects#734] [Help]
Hi!
First, thank you for this amazing plugin!
I start this discussion because I have a problem with one filter
I have a list of all my projects
a data field on each project note contains a list of links to my weekly review note to show me if I scan this project 
for example
```
Revue:
  - "[[KMS_jeremy/GTD/GTD-Review/WR 2023-S42.md]]"
  - "[[KMS_jeremy/GTD/GTD-Review/WR 2023-S43.md]]"
``` 
I want to see only the note where the review of week 43 is not done
So I try to set this filter:
revue as none of [[KMS_jeremy\GTD\GTD-Review\WR 2023-S44]]
but when I press tab ... [[KMS_jeremy\GTD\GTD-Review\WR 2023-S44]] is erase from the filter
where I'm wrong ?
Thanks

**Comment by Acylation** [comment]
Oh sorry that I've applied a stronger validation on the condition input. I've fixed this via #724 and just released v1.17.1. You can try it again in the new version.

A reminder that, please keep the slash and the suffix in consistent with note contents.
